src/pages/documentation/mobile-core/signal/event-reference.md

```diff
@@ -0,0 +1,91 @@
+# Signal event reference
+
+## Events handled
+
+The following events are handled by the Signal extension:
+
+### Rules Engine response content
+
+This event is used by the Signal extension to queue and send network calls that correspond to the [Sync PII Consequence](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details#sync-pii-consequence) or postback message. FIX LINK
+
+This event is triggered by the Rules Engine when the events and conditions are met.
+
+#### Event details
+
+| **Event Type** | **Event Source** | **Paired** | **Paired Event** |
+| :--- | :--- | :--- | :--- |
+| com.adobe.eventType.rulesengine | com.adobe.eventSource.responseContent | No | - |
+
+#### Data payload definition
+
+The following key-value pairs in this event correspond to postback and sync PII:
+
+| Key | Friendly name | Type | Optional | Description |
+| :--- | :--- | :--- | :--- | :--- |
+| `triggeredconsequence` | Description URL | Map | Yes | Triggered Consequence details in the map.    For the detailed description of each rule consequence, see the following pages:  [Postback consequence](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details#postback-consequence) [Sync PII consequence](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details#sync-pii-consequence) [Open URL consequence](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine/rules-engine-consequence-details#open-url-consequence) |
+
+FIX ABOVE LINKS
+
+#### Event data example
+
+The following message is an example of a postback:
+
+```json
+{
+    "id": "9d40f5665d5bdbe96dcb3a24f4e4fe98d686a602",
+    "type": "pb",
+    "detail": {
+        "templateurl": "https://www.endpoint.com/post/{%urlenc(~sdkver)%}",
```

Comment:
`templateUrl` from original docs differ from the one here
`"templateurl"   : "https://www.endpoint.com/post/<div data-gb-custom-block data-tag="urlenc"></div>",`